cities_indicators/indicators/non_tree_cover_by_land_use_gee.py

```diff
@@ -0,0 +1,178 @@
+from ..layers.non_tropical_tree_cover import NonTropicalTreeCover
+from ..layers.land_cover_class import LandCoverClass
+import ee
+import geemap
+import pandas as pd
+import math
+
+
+class NonTreeCoverByLandUseGEE:
```

Comment:
@weiqi-tori I think this there's a couple things in here from your other PR that don't seem relevant to this PR

Reply:
The [feature/natural_areas](https://github.com/wri/cities-cif/tree/feature/natural_areas) branch is based on the [feature/non_tree_cover_by_land_use](https://github.com/wri/cities-cif/tree/feature/non_tree_cover_by_land_use) branch. Since the previous branch/PR has not been merged, the file changes also showed up here.

Reply:
I changed the PR to be based on non_tree_cover_by_land_use